DOCPR-663: add section on Diátaxis to the style guide (#44)
canonical/praecepta: en/index.md

to the left, and presented here as a single page to aid searching.

## Diátaxis

Use Diátaxis to structure product documentation. A documentation page should
fit within one of the following categories:

- Tutorial
- How-to
- Explanation
- Reference

> Each category serves a different user need and is written in a specific style.

Reviewer: are they really written in a 'specific style'? I think that could be misinterpreted. My more fuzzy suggestion
```suggestion
Each category serves a different user need - it is useful to understand the needs being addressed as these may influence the documentation approach. 
```

to the left, and presented here as a single page to aid searching.

## Diátaxis

Use Diátaxis to structure product documentation.
Diátaxis provides rules that help you identify the type of document that you are writing.
Document types include:

- Tutorial
- How-to guide
- Reference
- Explanation

> Each type serves a different user need — it is useful to understand the needs being addressed as these will influence the documentation approach. 

Reviewer: ```suggestion
Each type serves a different user need — it is useful to understand the needs being
addressed as these will influence the documentation approach. 
```

## Diátaxis

Use Diátaxis to structure product documentation.
Diátaxis provides rules that help you identify the type of document that you are writing.

> Types of document include:

Reviewer 2: Here is a suggestion, but feel free to ignore it :)

```suggestion
Document types include:
```

- Tutorial
- How-to guides

> - Explanations

Reviewer 2: I think it should be "Explanation" (singular)?

Author: Sure. I will change this.

Author: Done.
